src/Nethermind/Nethermind.TxPool.Test/TxPoolTests.Blobs.cs

```diff
@@ -0,0 +1,396 @@
+                .WithMaxPriorityFeePerGas(1.GWei() + UInt256.One)
+                .WithNonce(UInt256.Zero)
+                .SignedAndResolved(_ethereumEcdsa, TestItem.PrivateKeyB).TestObject;
+
+
+            _txPool.SubmitTx(feeTooLow1559Tx, TxHandlingOptions.None).Should().Be(AcceptTxResult.FeeTooLow);
+            _txPool.SubmitTx(feeTooLowBlobTx, TxHandlingOptions.None).Should().Be(AcceptTxResult.Accepted);
+        }
+
+        [Test]
+        public void should_add_blob_tx_and_return_when_requested([Values(true, false)] bool isPersistentStorage)
+        {
+            TxPoolConfig txPoolConfig = new() { Size = 10, PersistentBlobStorageEnabled = isPersistentStorage };
+            BlobTxStorage blobTxStorage = new(new MemDb());
+            _txPool = CreatePool(txPoolConfig, GetCancunSpecProvider(), txStorage: blobTxStorage);
+            EnsureSenderBalance(TestItem.AddressA, UInt256.MaxValue);
+
+            Transaction blobTxAdded = Build.A.Transaction
+                .WithShardBlobTxTypeAndFields()
+                .WithMaxFeePerGas(1.GWei())
+                .WithMaxPriorityFeePerGas(1.GWei())
+                .WithNonce(UInt256.Zero)
+                .SignedAndResolved(_ethereumEcdsa, TestItem.PrivateKeyA).TestObject;
+
+            _txPool.SubmitTx(blobTxAdded, TxHandlingOptions.None).Should().Be(AcceptTxResult.Accepted);
+            _txPool.TryGetPendingTransaction(blobTxAdded.Hash!, out Transaction blobTxReturned);
+
+            blobTxReturned.Should().BeEquivalentTo(blobTxAdded);
+
+            blobTxStorage.TryGet(blobTxAdded.Hash, out Transaction blobTxFromDb).Should().Be(isPersistentStorage); // additional check for persistent db
+            if (isPersistentStorage)
+            {
+                blobTxFromDb.Should().BeEquivalentTo(blobTxAdded, options => options
+                    .Excluding(t => t.SenderAddress) // sender is not encoded/decoded...
+                    .Excluding(t => t.GasBottleneck) // ...as well as GasBottleneck...
+                    .Excluding(t => t.PoolIndex));   // ...and PoolIndex
+            }
+        }
+
+        [Test]
+        public void should_not_throw_when_asking_for_non_existing_tx()
+        {
+            TxPoolConfig txPoolConfig = new() { Size = 10 };
+            BlobTxStorage blobTxStorage = new(new MemDb());
+            _txPool = CreatePool(txPoolConfig, GetCancunSpecProvider(), txStorage: blobTxStorage);
+
+            _txPool.TryGetPendingTransaction(TestItem.KeccakA, out Transaction blobTxReturned).Should().BeFalse();
+            blobTxReturned.Should().BeNull();
+
+            blobTxStorage.TryGet(TestItem.KeccakA, out Transaction blobTxFromDb).Should().BeFalse();
+            blobTxFromDb.Should().BeNull();
+        }
+
+        [TestCase(999_999_999, false)]
+        [TestCase(1_000_000_000, true)]
+        public void should_not_allow_to_add_blob_tx_with_MaxPriorityFeePerGas_lower_than_1GWei(int maxPriorityFeePerGas, bool expectedResult)
```

Comment:
Because Geth is doing that 😄 And to limit number of possible replacements - blob transactions requires 100% fee bump to be replaced, but if it will start from 1 wei, it will still be possible DoS attack vector

Peter's words from https://github.com/ethereum/go-ethereum/pull/26940:
`Replacements are expensive. Given their size, propagating a replacement blob transaction to an existing one should be aggressively discouraged. Whilst generic transactions can start at 1 Wei gas cost and require a 10% fee bump to replace, we suggest requiring a higher min cost (e.g. 1 gwei) and a more agressive bump (100%).`